tests/k8s-azure/manifest/linux.json

```diff
@@ -1,47 +1,47 @@
 {
-  "apiVersion": "vlabs",
-  "location": "",
-  "properties": {
-    "orchestratorProfile": {
-      "orchestratorType": "Kubernetes",
-      "orchestratorVersion": "1.12.0-alpha.1",
-      "kubernetesConfig": {
-          "useCloudControllerManager": true,
-          "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.13.0-alpha.0",
-          "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.13.0-alpha.0",
-          "networkPolicy": "none",
-          "apiServerConfig": {
-            "--enable-admission-plugins": "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,AlwaysPullImages"
-          }
-      }
-    },
-    "masterProfile": {
-      "count": 1,
-      "dnsPrefix": "{dnsPrefix}",
-      "vmSize": "Standard_F2"
-    },
-    "agentPoolProfiles": [
-      {
-        "name": "agentpool1",
-        "count": 2,
-        "vmSize": "Standard_F2",
-        "availabilityProfile": "AvailabilitySet",
-        "storageProfile": "ManagedDisks"
-      }
-    ],
-    "linuxProfile": {
-      "adminUsername": "k8s-ci",
-      "ssh": {
-        "publicKeys": [
-          {
-            "keyData": "{keyData}"
-          }
-        ]
-      }
-    },
-    "servicePrincipalProfile": {
-      "clientID": "{servicePrincipalClientID}",
-      "secret": "{servicePrincipalClientSecret}"
+    "apiVersion": "vlabs",
+    "location": "",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "1.12",
+            "kubernetesConfig": {
+                "useCloudControllerManager": true,
+                "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.13.0-alpha.3",
+                "customHyperkubeImage": "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.13.0-alpha.3",
+                "networkPolicy": "none",
+                "apiServerConfig": {
+                    "--enable-admission-plugins": "NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,AlwaysPullImages"
+                }
+            }
+        },
+        "masterProfile": {
+            "count": 1,
+            "dnsPrefix": "{dnsPrefix}",
+            "vmSize": "Standard_F2"
+        },
+        "agentPoolProfiles": [
+            {
+                "name": "agentpool1",
+                "count": 2,
+                "vmSize": "Standard_F2",
+                "availabilityProfile": "VirtualMachineScaleSets",
```

Comment:
We can add another test to verify vmss, for here we still need to test vms.

Reply:
Good catch. This is from original PR, forgot to rollback.